libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestClient.java

```diff
@@ -395,6 +395,8 @@ public RestResponse sendSync(RestRequest restRequest, Interceptor... interceptor
 
     /**
      * All immutable information for an authenticated client (e.g. username, org ID, etc.).
+     *
+     * TODO revisit this class - some of information is NOT immutable e.g. sids
```

Comment:
The description for the class mentions "immutable information" like username and org id.
But it also olds SIDs which can change during a refresh :-(